keyboards/ergodox_infinity/ergodox_infinity.h

```diff
@@ -151,7 +149,7 @@ inline void ergodox_led_all_set(uint8_t n) {
     { B80,   B81,   B82,   B83,   B84 }  \
 }
 
-#define LAYOUT_ergodox KEYMAP
-#define LAYOUT_ergodox_pretty KEYMAP_PRETTY
+#define KEYMAP LAYOUT_ergodox
```

Comment:
Why do we need to #define KEYMAP to this?

Reply:
We don't, this was a "quick and dirty" change, and I didn't want to change all of the keymaps.